### PR TITLE
Corrected Javadocs name, title

### PR DESCRIPTION
**Pull request by MihirKohli**
### Description

Name & Title issue resolved 
https://github.com/SkriptLang/Skript/issues/5984

---
**Target Minecraft Versions:** 
**Requirements:** 
**Related Issues:**

### CODE REVIEW

File: .github/.DS_Store
Comment (AyhamAl-Ali) [inline .github/.DS_Store:1]: Should be removed.

File: .DS_Store
Comment (AyhamAl-Ali) [inline .DS_Store:1]: Should be removed.

File: .github/assets/icon.png
Comment (AyhamAl-Ali) [inline .github/assets/icon.png:1]: Should be 64x64px

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]
Thank you for your contribution ⚡

Just few little things

File: .gitignore [hunk 1]
```diff
@@ -223,3 +223,6 @@ gradle-app.setting
 # TODO remove this in the future after some time since https://github.com/SkriptLang/Skript/pull/4979
 # This ensures developers don't upload their old existing test_runners/ folder.
 test_runners/
+
+## Mac MetaData
+.DS_Store
```
Comment (AyhamAl-Ali) [inline .gitignore:228]: This now will ignore all DS files even nested ones
```suggestion
**/.DS_Store
```

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (MihirKohli) [reply .gitignore]: Changes Implemented 
https://github.com/SkriptLang/Skript/compare/master...MihirKohli:Skript:master?expand=1
Thank you for your guidance

**Review by AyhamAl-Ali — APPROVED** [review]
Well done ⚡

File: .gitignore [hunk 2]
```diff
@@ -223,3 +223,6 @@ gradle-app.setting
 # TODO remove this in the future after some time since https://github.com/SkriptLang/Skript/pull/4979
 # This ensures developers don't upload their old existing test_runners/ folder.
 test_runners/
+
+## Mac MetaData
+**/.DS_Store
```
Comment (TheLimeGlass) [inline .gitignore:228]: New line at end of file.
```suggestion
**/.DS_Store

```

**Review by TheLimeGlass — APPROVED** [review]

**Review by Moderocky — APPROVED** [review]

**Pull request merged**